### ISSUE TITLE
window is not defined [Building for Cloudflare Workers]

### ISSUE THREAD

**Issue by hrgui** [brillout/vite-plugin-ssr#130]
When deploying to Cloudflare Workers, the following error occurs

```ts
Error: Something went wrong! Status: 400 Bad Request, Details {
  "result": null,
  "success": false,
  "errors": [
    {
      "code": 10021,
      "message": "Uncaught ReferenceError: window is not defined\n  at line 1\n  at line 1\n  at line 1\n  at line 1 in n\n  at line 6\n  at line 6\n  at line 1 in n\n  at line 1\n  at line 1 in n\n  at line 33\n"
    }
  ],
```

This is because: https://github.com/brillout/vite-plugin-ssr/blob/master/vite-plugin-ssr/page-files/getPageFiles.shared.ts#L12 is called in the runtime of the worker when doing `wrangler publish`.

```ts
function assertNotAlreadyLoaded() {
  // The functionality of this file will fail if it's loaded more than
  // once; we assert that it's loaded only once.
  const alreadyLoaded = Symbol()
  const globalObject: any = isNodejs() ? global : window
  assert(!globalObject[alreadyLoaded]) // this is where it breaks
  globalObject[alreadyLoaded] = true // this also is where it breaks
}
```

The workaround I did to make this work is make isNodejs() evaluate to true,  then I can deploy to CF Workers.

**Comment by brillout** [comment]
I see what's going on, let me write a fix.

**Comment by brillout** [comment]
Fix released in `0.2.7`. I didn't test the fix, but it should work.

Sorry about the regression. If a regression happens again I'll start using miniflare to make sure there is no regression.

**Comment by hrgui** [comment]
Tested the test app that I have with 0.2.7, looks like it's working. I'm able to publish now.
